src/operator/nn/mkldnn/mkldnn_deconvolution.cc

```diff
@@ -216,11 +329,9 @@ void MKLDNNDeconvolutionBackward(const nnvm::NodeAttrs& attrs, const OpContext &
       bwdData_pd.src_primitive_desc());
   if (req[deconv::kData]) {
     auto weight_mem = GetWeights(inputs[deconv::kWeight + 1],
-                                 bwdData_pd.weights_primitive_desc(),
-                                 param.num_group);
+        bwdData_pd.weights_primitive_desc(), param.num_group);
```

Comment:
why do you change the indent here? the original indent follows the coding style in the mxnet.

Reply:
OK. will roll back